### DISCUSSION TITLE
Turning off the display

### DISCUSSION THREAD

**Discussion by Messier46** [Rem0o/FanControl.Releases#1968] [Q&A] [answered]
So I just recently started using Fan Control and I really like it. The only issue I am having is my displays will not turn off when running Fan Control. I wish the system would be able to go into sleep automatically but I understand that since the program constantly checking sensors it can't do that. Is there a way to have the program run and still have the display turn off automatically.

**Comment by JPKeith15** [comment]
My PC goes to sleep just fine with Fan Control running. The only issue I'm having is that when it wakes up from sleep Fan Control has an issue loading the user configuration and the sensors are showing up NaN. Have to restart the computer to fix it.

**Comment by Messier46** [comment]
If it should be able to go to sleep, then I'm not sure why it's keeping my computer awake. Fan Control was in my tray, and I checked 'powercfg /requests', which is what I usually used to check for background processes. It came up with nothing. Is there something else I should check?

**Reply by Rem0o** [reply]
Are you sure it is FanControl which prevents your monitor from turning off? By the way, monitor turning off and sleep are two different things, the 'powercfg /requests' command only applies to sleep.  Tested this on multiple computers and the monitor turns off just fine.

**Comment by Messier46** [comment]
I found the problem but I'm not sure on a good solution. When I try to connect to my Kraken AIO with either the LHM controller sensor or the Nzxt Kraken community plugin, it keeps my computer from turning off the display and going to sleep. Without either of these turned on my computer turns off its display and goes to sleep like normal.

I know the community plugin isn't necessary anymore, but I figured I would test it.

**Comment by Messier46** [comment, marked as answer]
So I found the solution and the reason feels arbitrary. I deleted an Alienware framework program that was just left on my computer from when I was testing the Alienware monitor program out on this computer. Now the display turn off and sleep functionality works perfect. It wasn't even the full Alienware program so it feels really odd that it would cause conflict.
